nifi-commons/nifi-utils/pom.xml

```diff
@@ -40,5 +40,10 @@
             <artifactId>nifi-api</artifactId>
             <version>1.7.0-SNAPSHOT</version>
         </dependency>
+        <dependency>
```

Comment:
Jackson is used in more packages than Gson, so I think you should switch over to that unless you have a compelling reason.

Reply:
I feel more convenient using Gson. I think it should be fine in this case since we are just trying to find if there are JSON syntax issues and not using some core serialization/deserialization that JSON libraries (Jackson, etc.,) offer.

Reply:
There are some things to consider when using Gson over Jackson and in general for this PR, this is related to the point I mentioned on a different PR about where to put a "well-formed JSON" validator:

1) Performance (https://blog.takipi.com/the-ultimate-json-library-json-simple-vs-gson-vs-jackson-vs-json/) 
2) The bundle already has Jackson (such as the Mongo bundle)
3) Increased NAR size (Gson is only ~23K so minor but 84 current bundles use nifi-utils and only 1-2 were updated to use the validator)

@zenfenan I agree that on the surface, Gson takes care of this issue conveniently and with a lightweight footprint. However because of the reasons above, it seems this could be problematic, and we should consider a different approach regardless of convenience.

Reply:
@mattyb149 I agree. 

@MikeThomsen Can you cover this in NIFI-5271 i.e. scrapping GSON and making use of Jackson?

Reply:
Yeah, NP and it'll make my life easier with the L&N.